.travis.yml

```diff
@@ -36,4 +44,6 @@ install:
   - export PATH=$PWD/travis-phantomjs:$PATH
 
 script:
-  - ember try $EMBER_TRY_SCENARIO test
+  # Usually, it's ok to finish the test scenario without reverting
+  #  to the addon's original dependency state, skipping "cleanup".
+  - ember try:one $EMBER_TRY_SCENARIO test --skip-cleanup
```

Comment:
Cool, this is the kind of gotcha that matters, too:

> The gotcha with --skip-cleanup is that if CI is also deploying, it could deploy with the last scenario run. Probably doesn't apply to many addons, though I know some deploy demo sites, but worth noting.